src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -141,6 +171,38 @@ namespace Google.Example.Library.V1.Snippets
             // End snippet
         }
 
+        public async Task GetShelfAsync_RequestObject()
+        {
+            // Snippet: GetShelfAsync(GetShelfRequest,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            GetShelfRequest request = new GetShelfRequest
+            {
+                Name = new ShelfName("[SHELF_ID]").ToString(),
```

Comment:
Hmm, many are (e.g. line 467).
I suspect this hasn't been defined as a resource-name in the test GAPIC configuration. Very possibly deliberately to test that it works properly.